lib/active_fedora/file.rb

```diff
@@ -34,7 +33,7 @@ class File
     # @param [Hash] _options
     # @yield [self] Yields self
     # @yieldparam [File] self the newly created file
-    def initialize(parent_or_url_or_hash = nil, path = nil, _options = {}, &_block)
+    def initialize(parent_or_url_or_hash = nil, _path = nil, _options = {}, &_block)
```

Comment:
@jcoyne would now be the correct time to remove the path parameter since it is not used?